R/trace.R

```diff
@@ -11,6 +11,11 @@
 #'   larger context, for example in tests or inside an RMarkdown
 #'   document where you don't want all of the knitr evaluation mechanisms
 #'   to appear in the backtrace.
+#' @param trim The number of parent contexts to trim from the bottom
+#'   of the backtrace, starting from the rightmost leaf (the newest
+#'   call frame in the tree). Contexts containing several frames
+#'   (these appear as sibling nodes in the backtrace tree) are fully
+#'   trimmed from the backtrace.
```

Comment:
It would be helpful if you could briefly explain when you'd want to set this, and how you find the correct value.

Reply:
I have renamed `to` to `top` and `trim` to `bottom`. Both arguments now take frame environments. This way it's more consistent and easier to document. I just mention that `bottom` can also be an integer passed to `caller_env()`, then we don't need to mention siblings. Documentation reviewed accordingly with a little more motivation.